### ISSUE TITLE
Mark notifications as read only when loading succeeds

### ISSUE THREAD

**Issue by rugk** [thermatk/FastHub#19]
**FastHub-Libre Version: 4.5.5**  
**Android Version: 7.1.2 (SDK: 25)**  
**Device Information:** ...

---

Sometimes I have the problem that I open a notification and it shows me a black screen with loading error (just some network issue, so I would usually just retry it), but in the background (with android notification) it marks the notification as read on github and in the app, so when I press "back" to repeat the losing I just cannot find the entry anymore.

**Comment by thermatk** [comment]
Hi!
The whole notification thing got rewritten in the new version, so when it will finally be available on F-Droid, might be fixed actually

You can list all notifications, not only those unread, in the tab "ALL" to the right, so nothing should be lost 👍

**Comment by rugk** [comment]
>  so nothing should be lost 

Yeah but with that (good) grouping there, it is hard to find it back.
